packages/p2p-v2/src/components/Modals/PaymentMethods/AddEditPaymentMethodErrorModal/AddEditPaymentMethodErrorModal.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import ReactModal from 'react-modal';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import { customStyles } from '../../helpers';
+import './AddEditPaymentMethodErrorModal.scss';
+
+type TAddEditPaymentMethodErrorModalProps = {
+    errorMessage: string;
+    isOpen: boolean;
+    onConfirm: () => void;
+};
+
+const AddEditPaymentMethodErrorModal = ({ errorMessage, isOpen, onConfirm }: TAddEditPaymentMethodErrorModalProps) => {
```

Comment:
```suggestion
const PaymentMethodErrorModal = ({ errorMessage, isOpen, onConfirm }: TPaymentMethodErrorModalProps) => {
```
`PaymentMethodErrorModal` naming imo is more clear than `AddEditPaymentMethodErrorModal`

Reply:
I think the initial naming is fine, since IMO its more specific for the adding and editing payment method sides, and not generalising payment method